archiver.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Archiver
+  attr_accessor :file
+
+  @modes = %i[reader writer both]
+
```

Comment:
Some extra empty lines can be taken by Ruby as bad practice, try to install the [Rubocop](https://docs.rubocop.org/rubocop/1.23/index.html) gem. It will help a lot to release about good and bad practices.

Reply:
Hello Diego! Thanks for the feedback! :star:
I love rubocop and always lint my code using the rubocop -A command! 
As you can see rubocop asks for that empty line 😵‍💫 
![image](https://user-images.githubusercontent.com/97312839/149028797-bdf906bb-99a1-4796-be5b-740472109f6e.png)